tox.ini

```diff
@@ -1,32 +1,36 @@
 [tox]
-envlist = dbt1.5, dbt1.6, dbt1.7, dbt1.8, dbtlatest
+envlist = dbt1.6, dbt1.7, dbt1.8, dbt1.9, dbtlatest
 
 [testenv:dbt{1.5,1.6,1.7,1.8,latest}]
 basepython = python3.10
 deps =
     pytest
+    pytest-asyncio
     pytest-cov
     pandas
     duckdb
     httpx
-    dbt1.5: dbt-duckdb==1.5.*
+    {dbt1.8,dbtlatest}: mcp>=1.0.0
```

Comment:
is it intentional to pick only `dbt1.8` and `dbtlatest` to save time?
or it should be between `dbt1.8` and `dbtlatest`